### DISCUSSION TITLE
Expound on backend architecture

### DISCUSSION THREAD

**Discussion by avosa** [home-assistant/architecture#744] [General] [answered]
I'm interested in learning the architecture of Home Assignment backend services/microservices and their communication (networking). And design/creteria usually folowed when either scaling or adding new feature. Any information/clarity on that will be very welcome.

**Comment by frenck** [comment, marked as answer]
Hi there @avosa 👋 

We use this repository to discuss architectural changes, not to provide support.
Please checkout of our developer documentation here:

https://developers.home-assistant.io

Or join our very active Discord chat, which has a bunch of developer-related channels that can help you out.

https://www.home-assistant.io/join-chat

../Frenck
